### ISSUE TITLE
Failed to write new FST after double bootup

### ISSUE THREAD

**Issue by kierio04** [TASLabz/dolphin#120]
Sometimes Dolphin will not want to boot up, so I will go to open it a second time. Usually, it will then boot up, but gives a "IOS_FS: Failed to write new FST" error.

**Comment by ximk** [comment]
Are you sure this is not just an issue for your PC? I can't replicate this (although I've never had issues having Dolphin boot up, which might be playing into it).

**Comment by kierio04** [comment]
Yeah it's entirely possible it's PC-related. I've also not had the issue many times since I opened this issue. I reckon put wontfix label on until someone else comes across the error, because it does definitely happen, but seems to not be a huge issue.

**Comment by ximk** [comment]
Going on stasis now until someone else can reproduce.

**Comment by malleoz** [comment]
This is likely OS-specific. The error is generated in `FS.cpp` when file IO fails (either the file could not be opened, or `fwrite` failed).

**Comment by vabold** [comment]
This sounds like the first Dolphin instance has the file opened while the second one tries to access it.
